api/src/middleware/meetingsRouter.ts

```diff
@@ -105,6 +105,9 @@ meetingsRouter.post('/:id/notes', async (req, res, next) => {
     );
     return;
   }
+
+  req.io.to(`meeting:${meetingId}`).emit('meeting_note:created', meetingNote);
```

Comment:
Why is `req` used instead of `res` here?

Reply:
We're using req here because it's trying to send/emit a message out to the specified room, it's not responding to a request.

Reply:
That makes sense.  `io` was also placed in the `req` object in the `meetingsService`.